elixir_server/mix.exs

```diff
@@ -0,0 +1,33 @@
+defmodule ElixirServer.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :elixir_server,
+      version: "0.1.0",
+      elixir: "~> 1.13",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger, :plug_cowboy, :plug, :poison,:tesseract_ocr],
+      mod: {ElixirServer.Application, []}
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:plug, "~> 1.5"},
+      {:poison, "~> 3.1"},
+      {:plug_cowboy, "~> 1.0"},
+      {:tesseract_ocr, "~> 0.1.5"}
+      # {:dep_from_hexpm, "~> 0.3.0"},
+      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
```

Comment:
remove these comments.
In fact, you can delete all comments in this file.

Reply:
Nope if some want to add something they can be helpful